index.html

```diff
@@ -422,6 +507,7 @@ <h2 class="ui header">Separate glyphs</h2>
   }
 
   $('#glyphs').empty()
+  $('#glyphs').hide()
```

Comment:
Unfortunately this breaks the `xAdvance` limit line (the red vertical bar) because then the glyph table is not shown yet when calculating the offset, which results in a `offsetLeft = 0`. For me, this does not speed up the glyph table generation a lot anyway so I would be inclined to remove the `hide()` call, WDYT @charno ?

Reply:
Ah yes, I didn't see the problem with the xAdvance.

For me in firefox on linux this speeds up the import greatly, from about 8 to 1.5 seconds for https://raw.githubusercontent.com/adafruit/Adafruit-GFX-Library/master/Fonts/Org_01.h.

I just tested with chrome, and there it's basically an instant operation independent if glyphs are shown or not. For a bigger font (Org_01 with added glyphs until 0xfff) it's a big difference also for chrome, but I guess that's very much an edge-case.

It's a small change, for me it's fine if it's removed and I might have a look later